pkg/controllers/interface_filter_test.go

```diff
@@ -0,0 +1,73 @@
+package controllers
+
+import (
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+	v1 "kubevirt.io/api/core/v1"
+)
+
+var _ = Describe("FilterMultusNonDefaultInterfaces", func() {
+	const defaultName = "default"
+	const nonDefaultName = "non default"
+	const nonDefaultName2 = "non default2"
+
+	It("when interfaces and networks lists are nil", func() {
+		Expect(FilterMultusNonDefaultInterfaces(nil, nil)).To(BeEmpty())
+	})
+	It("when interfaces and networks lists are empty", func() {
+		Expect(FilterMultusNonDefaultInterfaces([]v1.VirtualMachineInstanceNetworkInterface{}, []v1.Network{})).To(BeEmpty())
+	})
+	It("when there is only default network", func() {
+		Expect(FilterMultusNonDefaultInterfaces([]v1.VirtualMachineInstanceNetworkInterface{createVmInterface(defaultName)}, []v1.Network{createDefaultNetwork(defaultName)})).To(BeEmpty())
+	})
+	It("when there is a non default interface", func() {
+		nonDefaultInterface := createVmInterface(nonDefaultName)
+		result := FilterMultusNonDefaultInterfaces([]v1.VirtualMachineInstanceNetworkInterface{nonDefaultInterface, createVmInterface(defaultName)}, []v1.Network{createDefaultNetwork(defaultName), createMultusNonDefaultNetwork(nonDefaultName)})
+		Expect(result).To(HaveLen(1))
+		Expect(result).To(ContainElements(nonDefaultInterface))
```

Comment:
maybe instead two lines we can directly compare to the expected struct ?
if so all over, unless you prefer the current way

Reply:
If I compare the expected slice the order is also compared, and `FilterMultusNonDefaultInterfaces` doesn't promise anything regarding the order.

Reply:
There is ConsistOf IIRC that check all are there, length equal (double check me please), order not

Reply:
done, thanks.